middlewares/metrics.go

```diff
@@ -19,13 +24,41 @@ type Metrics interface {
 // given Metrics implementation to expose and monitor Traefik metrics
 type MetricsWrapper struct {
 	Impl Metrics
+
+	//frontend/backend
```

Comment:
The comment should maybe be a bit more elaborate.

Also, space between slash and first word.